physics/README.md

```diff
@@ -0,0 +1,44 @@
+# Physics
+This discipline of science is quite different and even nascent with regards to the lab. However, we are open to contributions and collaborations.
```

Comment:
"nascent with regards to IvLabs" is better I guess

Reply:
Yes, make the edit if you can

Reply:
ok